src/RollImage.cpp

```diff
@@ -5113,7 +5126,15 @@ std::string RollImage::getDruid(std::string input) {
 	if (input.empty()) {
 		input = getFilename();
 	}
-	auto loc = input.find('_');
+	auto loc = input.find_last_of('/');
+	if (loc != string::npos) {
+		input = input.substr(loc+1, input.size());
+	}
+	loc = input.find_last_of("\\/");
+	if (loc != string::npos) {
+		input = input.substr(loc+1, input.size());
+	}
```

Comment:
This seems redundant?  I'm very tired (!), but I'm pretty sure that if the string has already been trimmed down to everything after the last `/`, then `\/` can't ever be matched (and I'm unclear on why we'd be looking for this anyway?  Apologies if I've misunderstood...

Reply:
This code is meant to handle Windows path strings, so the idea is that the previous lines won't have done any trimming prior to this because on a Windows machine, the path string won't have forward slashes. It's not clear to my why `"\\/"` matches Windows `\` path delimiters, and to be honest I haven't actually tested this on Windows yet, but on the other hand it doesn't break anything on non-Windows machines, so the code is at least generally harmless.

Reply:
I think the point is that `string::find_last_of` takes a sequence of characters, so `input.find_last_of("\\/")` will find the last of either `/` or `\\` (== `\`), so if only *nix is to be supported the second block is redundant, but if this is to support Windows as well, the first block is redundant.

Reply:
Aha! Yes that is indeed how `string:: find_last_of` is supposed to work. I tested this with the first block (on a Mac) and it worked fine, so I think we can go with including just the first block.